settings-signatures.adoc

```diff
@@ -5,9 +5,6 @@ With {product-short}, you can have two default signatures per account, plus addi
 . Click *Signatures* in the left pane.
 .. To set a signature to be used while composing an email, draft your signature against the *Standard signature* field.
 .. To set a signature to be used while replying to or forwarding an email, draft your signature against the *Reply or forward signature* field.
-. To create additional signatures, click *Add a signature*.
```

Comment:
Any reason why this has been removed? I see this functionality still present in Modern UI.

Reply:
Because it is duplicated.   The lines 9-13 describes the process of Add a new signature.